server/server.go

```diff
@@ -268,6 +268,9 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 
 	// Init rand seed for randomBuf()
 	rand.Seed(time.Now().UTC().UnixNano())
+
+	variable.RegisterStatistics(s)
```

Comment:
Is the whole server struct needed to be sent for registration or is it just to have a reference to the needed part of the config/tls?